ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```
@@ -8,10 +8,13 @@ SSH_ARGS="-i ${CLUSTER_PROFILE_DIR}/jh_priv_ssh_key -oStrictHostKeyChecking=no -
 bastion=$(cat ${CLUSTER_PROFILE_DIR}/address)
 target_bastion=$(cat ${CLUSTER_PROFILE_DIR}/bastion)
 
-# Check if target bastion is in maintenance mode
-if ssh ${SSH_ARGS} -o ProxyCommand="ssh ${SSH_ARGS} -W %h:%p root@${bastion}" root@${target_bastion} 'test -f /root/pause'; then
-  echo "The cluster is on maintenance mode. Remove the file /root/pause in the bastion host when the maintenance is over"
-  exit 1
+if [[ ! -f "${SHARED_DIR}/assignment_id" ]]; then
+  target_bastion=$(cat ${CLUSTER_PROFILE_DIR}/bastion)
+  # Check if target bastion is in maintenance mode
+  if ssh ${SSH_ARGS} -o ProxyCommand="ssh ${SSH_ARGS} -W %h:%p root@${bastion}" root@${target_bastion} 'test -f /root/pause'; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)